tests/test_operations.py

```diff
@@ -51,29 +51,38 @@ def test_to_rna_2():  # Reference: https://biomodel.uah.es/en/lab/cybertory/anal
     assert oprimer_rna == "AUCGAUCGAUCG"
 
 
-def test_to_protein():  # Reference: https://biomodel.uah.es/en/lab/cybertory/analysis/trans.htm
+def test_to_protein_1(): 
+    # Reference:
+    # https://biomodel.uah.es/en/lab/cybertory/analysis/trans.htm
+    # https://en.vectorbuilder.com/tool/dna-translation.html
     oprimer = Primer("ATCGATCG")
-    oprimer_protein = oprimer.to_protein()
-    assert oprimer_protein == "ID"
 
+    # One-letter code translations
+    assert oprimer.to_protein() == "ID"
+    assert oprimer.to_protein(frame=2) == "SI"
+    assert oprimer.to_protein(frame=3) == "RS"
 
-def test_to_protein_2():  # Reference: https://en.vectorbuilder.com/tool/dna-translation.html
-    oprimer = Primer("ATCGATCG")
-    oprimer_protein = oprimer.to_protein(frame=2)
-    assert oprimer_protein == "SI"
-
+    # Three-letter code translations
+    assert oprimer.to_protein(multi_letter=True) == "lle-Asp"
+    assert oprimer.to_protein(frame=2, multi_letter=True) == "Ser-lle"
+    assert oprimer.to_protein(frame=3, multi_letter=True) == "Arg-Ser"
 
-def test_to_protein_3():  # Reference: https://en.vectorbuilder.com/tool/dna-translation.html
-    oprimer = Primer("ATCGATCG")
-    oprimer_protein = oprimer.to_protein(frame=3)
-    assert oprimer_protein == "RS"
 
-
-def test_to_protein_4():  # Reference: https://en.vectorbuilder.com/tool/dna-translation.html
+def test_to_protein_2():  
+    # Reference:
```

Comment:
`References`.

We might as well put the references right next to their golden tests and use `Reference`. That makes it even easier for future users to track the right reference.